docs/source/topics/proc_installing-codeready-containers.adoc

```diff
@@ -1,11 +1,57 @@
 [id="installing-codeready-containers_{context}"]
 = Installing {prod}
 
+{prod} is available as a portable executable for {rhel} and {msw}.
+On {mac}, {prod} is available using a guided installer.
+
 .Prerequisites
 
 * Your host machine must meet the minimum system requirements.
 For more information, see link:{crc-gsg-url}#minimum-system-requirements_gsg[Minimum system requirements].
 
 .Procedure
 
-. Download the link:{crc-download-url}[latest release of {prod}] for your platform and extract the contents of the archive to a location in your `_PATH_`.
+. Download the link:{crc-download-url}[latest release of {prod}] for your platform.
+
+. On {mac}, run the guided installer and follow the instructions.
+On {rhel} or {msw}, assuming the archive is in the [filename]*_~/Downloads_* directory, follow the procedure for your platform.
```

Comment:
Good point, I was unsure of using "assuming" myself. My reasoning for writing it this way is as follows, though: technically, we cannot control where a user downloads the file, but the *_Downloads_* directory is the default on every major platform unless configured otherwise. Since we can't be sure that the archive is downloaded there, we assume that it is and inform the reader of that assumption so that they can modify the commands appropriately if they downloaded it somewhere else.

Reply:
Good reasoning! I think it's reasonable to use `assuming` in this instance.